Web/starter_project_barbies/components/about/ProjectCard.tsx

```diff
@@ -0,0 +1,49 @@
+import { ProjectCardProps } from '@/types/AboutTypes'
+import Image from 'next/image'
+
+const ProjectCard = ({ project, isEven }: ProjectCardProps) => {
+  return (
+    <div
+      className={`flex flex-wrap justify-around ${
+        !isEven ? 'flex-row-reverse' : ''
+      }`}
+    >
+      <Image
+        className="flex-initial"
+        src={project.image}
+        width={700}
+        height={200}
+        alt="hakimhub"
```

Comment:
[very-nit] i think you don't need to condense the alt too much, you can just say Hakim Hub or sth